City-of-Helsinki/drupal-helfi-etusivu
# UHF-5915: Add filter pills styles

# [UHF-5915](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5915)
Add filter pills to search app

## What was done
* Added styles for showing the filter pills
* Bumped React app version to 0.5.0

## How to install

* Checkout branch
* Run `make fresh`
* Check that [news index](https://helfi-etusivu.docker.so/fi/admin/config/search/search-api/index/news) has news indexed
  * If not, try clicking "rebuild tracking information" and then "index now" 
* Run `composer require drupal/hdbt:dev-UHF-5915-filter-pills`
* Run `make drush-cr`

## How to test
* Navigate to news page: https://helfi-etusivu.docker.so/fi/uutiset  
* Try adding some selection to news filters. They should be reflected as pills under the dropdowns
* Clicking the pills should remove the selection
* Pills should look like the ones defined here: https://share.goabstract.com/e4d7b8d2-b559-4a80-b2ff-432ee2ab83e4?collectionLayerId=650cf13a-e1e7-49ae-821a-39c68be6c9dd&mode=build

## Designers review



## Other PRs


* [HDBT](https://github.com/City-of-Helsinki/drupal-hdbt/pull/394)
* [React app](https://github.com/City-of-Helsinki/helfi-etusivu-news-search/pull/7)
